--- Triathlon-Breeder.py	2009-11-28 15:28:04.000000000 +0100
+++ patches/Triathlon-Breeder.py.new	2009-12-10 01:30:54.000000000 +0100
@@ -125,7 +125,7 @@
                  learning_rate      = 0.5,
                  max_iterations     = 50,
                  bornBefore         = 0,
-                 trainAlg           = libfann.TRAIN_RPROP,
+                 trainAlg           = libfann.FANN_TRAIN_RPROP,
                  learning_momentum  = 0.0,
                  neurons            = [],
                  connectionType     = "Sparse"):
@@ -228,7 +228,7 @@
             if p < 0.25:
                 self.trainAlg = libfann.TRAIN_BATCH
             elif p < 0.5:
-                self.trainAlg = libfann.TRAIN_RPROP
+                self.trainAlg = libfann.FANN_TRAIN_RPROP
             elif p < 0.75:
                 self.trainAlg = libfann.TRAIN_INCREMENTAL
             else:
